<launch>
  <node name="pong" pkg="sawyer_pong" type="pong.py" output="screen"/>
  <node name="armcontrol" pkg="sawyer_pong" type="armcontrol.py"/>

</launch>
